types/src/front/lib/assistant.ts

```diff
@@ -92,7 +92,7 @@ export const GPT_4_TURBO_MODEL_CONFIG: ModelConfigurationType = {
 export const GPT_4O_MODEL_CONFIG: ModelConfigurationType = {
   providerId: "openai",
   modelId: GPT_4O_MODEL_ID,
-  displayName: "GPT-4o",
```

Comment:
I think we want just nothing instead of `-` (not a space) to stay consistent with OpenAI